.github/workflows/test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           token: ${{ secrets.GH_GITHUB_COM_TOKEN || github.token}}
       - run: |
           python -m pip install --upgrade pip
-          python -m pip install .[with-dependencies,postgres,ape]
+          python -m pip install .[with-dependencies,postgres]
```

Comment:
we no longer have a postgres dependency section. should we just add it to `with-dependencies`?